### DISCUSSION TITLE
Issue installing

### DISCUSSION THREAD

**Discussion by iep1000** [Homebrew/discussions#1359] [Getting started]
I'm running Catalina 10.15.7 on my Macbook Pro and trying to install Homebrew. Once I get it loaded into terminal, a pop up window appears that says "_The 'xcode-select' command requires the command line developers tools. Would you like to install the tools now?_" 

I select install, but then it tells me "_Can't install the software because it is not currently avaliable from the software update server._"

Do I need to update my computer to BigSur to get this to work?

**Comment by SMillerDev** [comment]
https://developer.apple.com/download will likely have a download for you. I believe the app store is just partially broken on Catalina.

**Comment by carlocab** [comment]
Updating to Big Sur will work. Otherwise, you can try installing the Command Line Tools for Xcode 12.4 from https://developer.apple.com/download/more/.
